app/models/piece.rb

```diff
@@ -93,4 +93,20 @@ def remains_on_board?(to_x:, to_y:)
       false
     end
   end
+
+  def successful_capture(to_x:, to_y:)
```

Comment:
Re: when the `capture` method would be called, I'd say it is **only** when:
- the moving piece's `valid_move?` method returns `true`
- **and** there is an **active** piece at the (`to_x:`, `to_y`) passed to `valid_move`
- **and** the active piece at (`to_x:`, `to_y`) is the **opposite color** of the moving piece